tests/ui/privacy/private-type-in-interface.stderr

```diff
@@ -46,11 +46,23 @@ error: type `Priv` is private
 LL | fn g() -> impl Tr2<m::Alias> { 0 }
    |           ^^^^^^^^^^^^^^^^^^ private type
 
+error: type `Priv` is private
+  --> $DIR/private-type-in-interface.rs:28:16
+   |
+LL | fn g() -> impl Tr2<m::Alias> { 0 }
+   |                ^^^^^^^^^^^^^ private type
```

Comment:
That's a weird almost-duplication and not even the right span, either before or after this change.

The only functional change should have been the generic arg one, and I don't see why checking consts and lifetimes would have an effect here, as there are only traits and types involved.

Any ideas?

Reply:
Looks like `visit_item_likes_in_module` includes `impl Trait` items, and `intravisit::walk_mod` does not.